draft-boucadair-add-deployment-considerations.txt

```diff
@@ -413,7 +413,9 @@ Internet-Draft    Discovery of Encrypted DNS Resolvers    September 2022
 
    Figure 4 shows various deployments where the CPE embeds a caching DNS
    forwarder.  Cases (b) and (d) involves a host (called legacy host)
-   that does not support DNR.
+   that does not support DNR.  In all cases, DDR is not applicable
+   because DDR requires the use of public IP addresses for proper
+   verification of ownership which the CPE will not be using.
```

Comment:
DDR is not applicable for private IPv4 addresses used but can be used with public IPv6 addresses used by the CPE.

Reply:
I'm not sure what you're suggesting here: it says DDR requires public addresses which the CPE will not be using in this section's case (proxied DNS through the CPE itself). IPv4 versus IPv6 is irrelevant; the only dimension that matters is public versus private.

Reply:
> This scope of this section is not restricted to private IPv4 addresses.

Agreed, hence the text saying that DDR doesn't apply here. I'm sorry, I'm still missing what change you'd like to see.

Reply:
I would like to see the following text removed : "In all cases, DDR is not applicable because DDR requires the use of public IP addresses for proper verification of ownership which the CPE will not be using."

CPE can be using a public IPv6 address and DDR is also applicable to this section as per the new section proposed by Dan.